modules/deploy/pages/deployment-option/cloud/configure-privatelink-private-service-connect-in-cloud-ui.adoc

```diff
@@ -0,0 +1,88 @@
+= Configure PrivateLink and Private Service Connect in the Cloud UI
+:description: Set up AWS PrivateLink and GCP Private Service Connect in the Redpanda Cloud UI.
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+NOTE: This guide is for configuring AWS PrivateLink and GCP Private Service Connect using the Redpanda Cloud UI. See xref:deploy:deployment-option/cloud/aws-privatelink.adoc[Configure AWS PrivateLink for Redpanda Cloud] or xref:deploy:deployment-option/cloud/gcp-private-service-connect.adoc[Configure GCP Private Service Connect for BYOC] if you want to set up these services using the API.
+
+The Redpanda AWS PrivateLink and GCP Private Service Connect services provide secure access to Redpanda Cloud from your own VPC. Traffic over PrivateLink and Private Service Connect do not go through the public internet because these connections are treated as their own private AWS and GCP services. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.
+
+Consider using the endpoint services if you have multiple VPCs and could benefit from a more simplified approach to network management:
```

Comment:
@0x5d @paulzhang97 Terminology question: AWS uses "endpoint services" whereas GCP seems to just use "service". I'll run this by the docs team as well, but do you have any thoughts on whether it's ok to refer to both collectively as "services" or "endpoint services"? I think I lean towards "endpoint service" since that more easily distinguishes it from a "Redpanda service" (such as HTTP Proxy).

Reply:
We should probably run this by GCP customers. If "endpoint services" is the term in AWS, I don't see anyone being confused over it if they're on AWS. However, I don't remember seeing this in GCP docs, which is my frame of reference (and likely other customers', like DnB or Moov).

For example, I'm having a hard time inferring what "endpoint services" would translate to here on the GCP side. Is it the service attachment? Is it the PSC endpoint?

Reply:
`Endpoint Service` is AWS term. How about `Consider using AWS PrivateLink or GCP Private Service Connect service`?

Reply:
@0x5d @paulzhang97 I've separated these out into two different docs. I think that makes the terminology a bit less confusing as I was trying to use one term/phrase to describe both PL and PSC.